canonical-web-and-design/snapcraft.io
# Build snaps list per store

## Done

- Show first store snaps view at /admin
- Show appropriate store at /admin/:store_id/snaps

## QA
- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/admin
- Check that the first store is selected and shows the snaps table
- Go to the first stores snaps link
- Check that the store is selected and shows the snaps table
